Packs/Veeam/Playbooks/Veeam_Start_Instant_VM_Recovery_Automatically.yml

```diff
@@ -0,0 +1,1180 @@
+        completeafterreplies: 1
+        completeaftersla: false
+        completeafterv2: true
+        retriescount: 2
+        retriesinterval: 360
+      to:
+    nexttasks:
+      Finish:
+      - "6"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 100b2818-b1e7-4886-8f4a-91be215bec53
+      iscommand: false
+      name: No Clean Restore Points Found
+      type: condition
+      version: -1
+    taskid: 100b2818-b1e7-4886-8f4a-91be215bec53
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 380
+        }
+      }
+  "34":
+    continueonerrortype: ""
+    id: "34"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "11"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Delete field from context.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      id: 9bb5d1ee-3248-499b-802a-6878ca2d4a2d
+      iscommand: false
+      name: Clear Context Data
+      script: DeleteContext
+      type: regular
+      version: -1
+    taskid: 9bb5d1ee-3248-499b-802a-6878ca2d4a2d
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": -1440
+        }
+      }
+  "35":
+    continueonerrortype: ""
+    id: "35"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ""
+      iscommand: false
+      max: 100
+      wait: 1
+    note: false
+    quietmode: 0
+    scriptarguments:
+      Instance:
+        complex:
+          root: inputs.Instance
+      backupObjectId:
+        complex:
+          root: inputs.backupObjectId
+    separatecontext: true
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 560a7e17-df26-42d3-80fc-5dfd0c8a059a
+      iscommand: false
+      name: Veeam - Start Instant VM Recovery Manually
+      playbookId: Veeam - Start Instant VM Recovery Manually
+      type: playbook
+      version: -1
+      description: ''
+    taskid: 560a7e17-df26-42d3-80fc-5dfd0c8a059a
+    timertriggers: []
+    type: playbook
+    view: |-
+      {
+        "position": {
+          "x": 730,
+          "y": 380
+        }
+      }
+  "36":
+    continueonerrortype: ""
+    id: "36"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ""
+      iscommand: false
+      max: 100
+      wait: 1
+    nexttasks:
```

Comment:
Please connect this task to `Done` or other task as needed.

Reply:
Done